dev/conductor/ui/lib/main.dart

```diff
@@ -36,19 +59,48 @@ class MyApp extends StatelessWidget {
       title: _title,
       theme: ThemeData(
         primarySwatch: Colors.blue,
+        scaffoldBackgroundColor: Colors.grey[800],
+        backgroundColor: Colors.black,
+        primaryColor: Colors.black,
+        iconTheme: const IconThemeData().copyWith(color: Colors.white),
+        fontFamily: 'Montserrat',
```

Comment:
Is this font already bundled with Flutter?

Reply:
Oh you are right, I need to import it. I removed this font. The default font looks good :)

Reply:
> > On Linux the tabs are not rendered properly:
> > ![Screenshot from 2021-09-28 16-47-51](https://user-images.githubusercontent.com/7856010/135180393-c7235936-4dc2-4845-8bf4-a986ff17f1fa.png)
> > Is this what you see on mac? I don't think this is a blocker necessarily to this PR, but certainly something we should come back and clean up (ideally this information would be presented prettier anyway)
> 
> I don't see the `HT` on Mac. It doesn't matter for now. I will eventually make a UI to present the state nicely.

Removing the "Montserrat" font fixed this :)